mlflow/models/evaluation/base.py

```diff
@@ -456,11 +485,99 @@ def _start_run_or_reuse_active_run(run_id):
     else:
         if run_id and active_run.info.run_id != run_id:
             raise ValueError(
-                "An active run exists, you cannot specify another run_id when " "evaluating."
+                "An active run exists, you cannot specify another run_id when evaluating."
             )
         yield active_run.info.run_id
 
 
+def _normalize_evaluators_and_evaluator_config_args(
+        evaluators,
+        evaluator_config,
+):
+    from mlflow.models.evaluation.evaluator_registry import _model_evaluation_registry
+
+    if evaluators is None:
+        evaluator_name_list = list(_model_evaluation_registry._registry.keys())
+        if evaluator_config is not None:
+            raise ValueError(
+                'If `evaluators` argument is None, `evaluator_config` argument must be None too.'
```

Comment:
Can we give users a hint about configuring the default evaluator? E.g. "To configure the default evaluator, specify evaluators="default" along with your evaluator_config".